otp.sh

```diff
@@ -19,13 +21,25 @@ for dir in "$idir"/lib/*/include "$idir"/bootstrap/lib/*/include; do
     includes="$includes -I $dir"
 done
 
+for dir in `find $idir/lib/*/src -name "*.hrl"` `find $idir/bootstrap/lib/*/src -name "*.hrl"`; do
+    includes_src="$includes_src\n$(dirname $dir)"
+done
+
+for dir in `echo -e "$includes_src" | uniq`; do
+    includes="$includes -I $dir"
+done
+
+cd "$idir"
```

Comment:
Change current directory to the `otp_src_*` directory so that the `building_otp` config value results in `true`.